### PR TITLE
generic: Convert incorrect generic/5.15 patches again

### PR DESCRIPTION
**Pull request by oliv3r**
OpenWRT's developer guide prefers having actual patches so they an be sent upstream more easily.

However, in this case, Adding proper fields also allows for `git am` to properly function. Some of these patches are quite old, and lack much traceable history.

This commit tries to rectify that, by digging in the history to find where and how it was first added.

It is by no means perfect and also shows some patches that should have been long gone.

### CODE REVIEW

**Comment by oliv3r** [comment]
@Ansuel maybe look again at https://github.com/openwrt/openwrt/pull/11600 as that could be used in the pipeline to prevent these mishaps :)

**Comment by mpratt14** [comment]
I would reword commit title to much simpler "add commit message to some patches"

**Comment by oliv3r** [comment]
> I would reword commit title to much simpler "add commit message to some patches"

Could, but I just re-used the same commit message I used last two times; so probably easiest to just re-use the existing one until we can prevent these from getting merged :)

**Comment by Ansuel** [comment]
@dangowrt ok to add the sob tag?

File: target/linux/generic/backport-5.15/733-v6.3-18-net-ethernet-mtk_eth_soc-add-support-for-MT7981.patch [hunk 1]
```diff
@@ -1,3 +1,13 @@
+From 7472476aef263bbf49b19d084875caddb7fe8f2a Mon Sep 17 00:00:00 2001
+From: Felix Fietkau <nbd@nbd.name>
+Date: Wed, 29 Mar 2023 11:38:30 +0200
+Subject: [PATCH] kernel: fix mtk_eth_soc throughput regressions on gigabit PHY
+ ports
+
+Fixes issues on MT7621 and MT7623 switch ports, and PHY ports on all SoC
+
+Signed-off-by: Felix Fietkau <nbd@nbd.name>
```
Comment (hauke) [inline target/linux/generic/backport-5.15/733-v6.3-18-net-ethernet-mtk_eth_soc-add-support-for-MT7981.patch:9]: This patch has a different commit message: 
https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net-next.git/commit/?id=f5d43ddd334b7c32fcaed9ba46afbd85cb467f1f

Comment (oliv3r) [reply target/linux/generic/backport-5.15/733-v6.3-18-net-ethernet-mtk_eth_soc-add-support-for-MT7981.patch:9]: I couldn't find the original, so I took it from the openwrt commit message.

I've pulled that patch instead and replaced it.

Would have been easier if we would 'get it right' from the get-go :)

Thanks @hauke

Comment (hauke) [reply target/linux/generic/backport-5.15/733-v6.3-18-net-ethernet-mtk_eth_soc-add-support-for-MT7981.patch]: Yes it would have been better if the commit description would have been added directly before the patch was added. 
When a patch is in the backports directory it should be in linux-next too, I was just looking there for the changes.
